evaluation/src/main/java/com/formulasearchengine/mlp/evaluation/GoldEntry.java

```diff
@@ -0,0 +1,77 @@
+package com.formulasearchengine.mlp.evaluation;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Created by Leo on 05.10.2016.
+ */
+public class GoldEntry {
+  private final String qID;
+  private final String oldId;
+  private final String fid;
+  private final String mathInputTex;
+  private final String title;
+  private final List<IdentifierDefinition> definitions;
+
+
+  public String getqID() {
+    return qID;
+  }
+
+  public String getOldId() {
+    return oldId;
+  }
+
+  public String getFid() {
+    return fid;
+  }
+
+  public String getMathInputTex() {
+    return mathInputTex;
+  }
+
+  public String getTitle() {
+    return title;
+  }
+
+  public List<IdentifierDefinition> getDefinitions() {
+    return definitions;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+
+    GoldEntry goldEntry = (GoldEntry) o;
+
+    if (!getqID().equals(goldEntry.getqID())) return false;
+    if (!getOldId().equals(goldEntry.getOldId())) return false;
+    if (!getFid().equals(goldEntry.getFid())) return false;
+    if (!getMathInputTex().equals(goldEntry.getMathInputTex())) return false;
+    if (!getTitle().equals(goldEntry.getTitle())) return false;
+    return getDefinitions().equals(goldEntry.getDefinitions());
+
+  }
+
+  @Override
+  public int hashCode() {
```

Comment:
I needed the equals to make the list comparison for the `definitions` work as expected, this implies a custom hashCode. Also it is the one generated by Idea.